### PR TITLE
Add citation file

### PR DESCRIPTION
**Pull request by Eimolgon**
Included names of the authors and citation style

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+authors:
+- family-names: "van Steen"
+  given-names: "Floyd"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Mirketou"
+  given-names: "Anastasia"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Acquaviva"
+given-names: "Marisa"
+- family-names: "Gonzalez"
```
Comment (manuGil) [inline CITATION.cff:12]: I think we should include all the members of the team as authors

**Review by manuGil — CHANGES_REQUESTED** [review]
Please make the changes I suggested

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+given-names: "Benjamin"
+title: "Collab 1"
+version: 1.0.0.0
+doi: 10.5281/zenodo.1234
+date-released: 2024-10-10
+url: "https://github.com/github-linguist/linguist"
+
```
Comment (FvanSteen) [inline CITATION.cff:19]: Please add the name of the repo as title.

File: CITATION.cff (hunk 1 above)
Comment (Eimolgon) [reply CITATION.cff:12]: They are included

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "van Steen"
+  given-names: "Floyd"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Mirketou"
```
Comment (amarketou) [inline CITATION.cff:7]: My name is Marketou, can you change it please

**Review by amarketou — COMMENTED** [review]
Kindly, Anastasia

File: CITATION.cff (hunk 2 above)
Comment (Eimolgon) [reply CITATION.cff:22]: Ready

File: CITATION.cff (hunk 3 above)
Comment (Eimolgon) [reply CITATION.cff]: Solved dear anastasia

**Review by FvanSteen — APPROVED** [review]

**Pull request merged**